otmonitor/rootfs/usr/share/otmonitor/html/navbar.inc

```diff
@@ -0,0 +1,67 @@
+<!-- *** NAVBAR ************************************************************ -->					
+<nav class="navbar navbar-fixed-top" role="navigation">
+	<div class="container-fluid">
+		<!-- Brand and toggle get grouped for better mobile display -->	
+		<div class="navbar-header">
+			<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+				<span class="sr-only">Toggle navigation</span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</button>
+			<a id="showMenu" class="navbar-brand" href="\">Opentherm Gateway</a>			
+		</div>
+			<!-- Collect the nav links, forms, and other content for toggling -->
+		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+			<ul class="nav navbar-nav navbar-right">
+				<li><a href="status.html">Status</a></li>
+				<li><a href="graph.html">Graph</a></li>
+				<li><a href="messages.html">Messages</a></li>
+				<li><a href="summary.html">Statistics</a></li>
+                <li><a href="thermostat.html" target="_blank">Thermostat</a></li>		
+				<li class="dropdown">
+					<a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Administration <span class="caret"></span></a>
+					<ul class="dropdown-menu">
+					<!--<li class="active"><a href="alert.html">Alert</a></li>
```

Comment:
Are these comments still useful?

Reply:
No not really, they are original code and very ugly :) I'll strip right after fixing the merge conflicts I've got myself in :D